### PR TITLE
Add helper script to optionally run kubectl/helm/helmfile from Docker

### PR DESCRIPTION
**Pull request by manics**
These can all be run natively. This script provides an optional preconfigured environment for anyone who doesn't want to install kubectl helm helmfile.

Takes advantage of https://github.com/IDR/kube-helm-docker/pull/1

### CODE REVIEW

File: run-with-docker.sh [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    exit 1
+fi
+
+KUBECONFIG=$(realpath $1)
+shift
+
+exec docker run -it --rm \
+    -v "$KUBECONFIG:/home/kube/.kube/config" \
+    -v "$PWD/..:/k8s:ro" -w /k8s/apps \
+    kube-helm-docker "$@"
```
Comment (joshmoore) [inline run-with-docker.sh]: Where is this image coming from?

Comment (manics) [reply run-with-docker.sh]: Race condition! Two cross-linked PRs https://github.com/IDR/kube-helm-docker/pull/1

Comment (manics) [reply run-with-docker.sh]: It should probably move to ome...

**Comment by sbesson** [comment]
Closing as we are not going to pursue the effort on the K8s cluster in the immediate future.

**Pull request closed without merging**